documentation/docs/steps/transportRequestUploadRFC.md

```diff
@@ -0,0 +1,105 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+* You have enabled RFC on the ABAP system.
+* You have a user account on the ABAP system where you have assigned the required roles for uploading via RFC.
+* You have created a transport request on the ABAP system, which is the target container of the upload.
+
+## RFC Client
+
+The RFC Client connects to your ABAP system using the [SAP NetWeaver RFC SDK](https://support.sap.com/en/product/connectors/nwrfcsdk.html).
+
+For more information, see [classical SAP connectivity technology RFC](https://help.sap.com/viewer/753088fc00704d0a80e7fbd6803c8adb/1709%20000/en-US/4888068ad9134076e10000000a42189d.html) for detailed information.
```

Comment:
Delete "for detailed information" at the end of the sentence.

Reply:
done